### DISCUSSION TITLE
malformed config.rasi file gets created after every boot

### DISCUSSION THREAD

**Discussion by tobip** [davatorium/rofi#1814] [(Possible) Issues]
Hi, after the last update of rofi, every time I boot up my machine, there appears a new file `~/.config/rofi/config.rasi` which will break functionality of rofi.

The content of this file is only one line:
`@import Manjaro`

rofi -version is `Version: 1.7.5+wayland1-42-g5d4a3e14 (makepkg)`
This is on Manjaro sway edition, running wayland.

When I run rofi this error appears:
![image](https://user-images.githubusercontent.com/3918330/219853756-da65d661-0058-463e-8450-8a753e1a57b1.png)

After I remove the forementioned file, the error does not occur any more and rofi works normally, until I reboot the machine and the file again gets created automatically by something.

**Comment by DaveDavenport** [comment]
rofi does not create any files. nor get launched at boot.
It must be something else on your system doing this.

**Reply by DaveDavenport** [reply]
I also think you are running https://github.com/lbonn/rofi and not this rofi.

**Reply by tobip** [reply]
I am not creating the file either!
How can I find out what does?

> I also think you are running https://github.com/lbonn/rofi and not this rofi.

I am coming from that page, it points directly to this forum.

**Reply by DaveDavenport** [reply]
your distribution, or you must have installed a tool that creates the file. 


I don't know what else to say?
This rofi does not create it own config file (it has no code for this).
The only tool we ship that does is rofi-theme-selector, but this is interactive and does not use ```import``` syntax.
Rofi also has nothing that (by the action of rofi) starts it self on bootup.

**Reply by DaveDavenport** [reply]
>  >    I also think you are running https://github.com/lbonn/rofi and not this rofi.
>
> I am coming from that page, it points directly to this forum.

While I don't think this issue has anything to do with the modifications lbonn made to rofi, I cannot be 100% sure as I don't know his code. You cannot expect support for any project that just happens to point to this forum. 
(hmm there are 577 pages on github that point to here :D)

**Reply by lbonn** [reply]
> > I also think you are running https://github.com/lbonn/rofi and not this rofi.
> 
> I am coming from that page, it points directly to this forum.

Ah, sorry about that, probably better to change it.

**Comment by lbonn** [comment]
@tobip I guess you are using the manjaro sway project. This seems to be the culprit, you can comment this line or maybe they have some kind of framework to handle configuration differently?

https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/etc/sway/definitions#L141

**Reply by DaveDavenport** [reply]
Some escaping  (in sway?) must go wrong, I don't think that line should cause the above error? 

tobip reported:
```css
@import Manjaro
```
I suspect it should be:
```css
@import "Manjaro"
```
this (from above link)
```bash
echo '@import "Manjaro"' > $HOME/.config/rofi/config.rasi'
```
should generate that.

@tobip  you can also try not deleting config.rasi , but replacing it with an empty file. (if you use that 'manjaro-sway' thing)

**Reply by tobip** [reply]
Thank you a lot, this does help!
And sorry for the troubles in this discussion board.

**Reply by DaveDavenport** [reply]
>  And sorry for the troubles in this discussion board.

no problem at all.
